tests/parametric/test_telemetry.py

```diff
@@ -463,7 +474,7 @@ class Test_Stable_Configuration_Origin(StableConfigWriter):
                     "logs_injection_enabled": "fleet_stable_config",
                     # Reporting for other origins than stable config is not completely implemented
                     # "runtime_metrics_enabled": "env_var",
-                    "profiling_enabled": "local_stable_config",
+                    "dynamic_instrumentation_enabled": "local_stable_config",
```

Comment:
I changed that because for PHP, profiler and tracer are 2 separated extensions, and currently, only the tracer sends configuration telemetry